tests/helpers/test_trigger.py

```diff
@@ -37,6 +39,34 @@
 from tests.common import MockModule, MockPlatform, mock_integration, mock_platform
 
 
+@pytest.mark.parametrize(
+    ("relative_key", "absolute_key"),
+    [
+        ("turned_on", "homeassistant.turned_on"),
+        ("_", "homeassistant"),
+        ("_state", "state"),
+    ],
+)
+def test_absolute_trigger_key(relative_key: str, absolute_key: str) -> None:
```

Comment:
since these can be tested indirectly trough the public methods, I think we can remove the test to avoid having tests for "internal" methods